deploy/helm_charts/envs/mixer_autopush.yaml

```diff
@@ -27,16 +27,6 @@ serviceGroups:
         memoryRequest: "8G"
         memoryLimit: "8G"
       useSearch: true
-  - recon:
```

Comment:
Not following why need to remove this? This is to ensure all the recon APIs are routed to the recon pods and not affecting other APIs.

Reply:
I was encountering an issue where recon apis were not being detected, looked for the recon flag in main.go and found out that recon is merged into main service. Not remembering that we still wanted separate traffic, I thought recon API group was meant to be removed as well. I reverted this change.

As for the recon api detection issue, will follow up on that separately.

Reply:
ack.